Scripts/Restriction/ranacompiler.py

```diff
@@ -367,9 +366,11 @@ def __init__(cls):
 # license.  Please see the LICENSE file that should have been included
 # as part of this package.
 #
-# This file is automatically generated - do not edit it by hand! Instead,
-# use the tool Scripts/Restriction/ranacompiler.py which in turn uses
-# Scripts/Restriction/rebase_update.py and Bio/Restriction/RanaConfig.py
+# THIS FILE IS AUTOMATICALLY GENERATED - DO NOT EDIT IT BY HAND!
+#
+# Instead, use the tool Scripts/Restriction/ranacompiler.py which in turn
+# uses Scripts/Restriction/rebase_update.py and Bio/Restriction/RanaConfig.py
+
```

Comment:
This line should be shortened to:
```python
# uses Scripts/Restriction/rebase_update.py
```

`RanaConfig.py` was deleted two or three releases ago.

Reply:
Good catch - I'll fix that on #2592, and then update this pull request accordingly.